examples/text.js

```diff
@@ -1,13 +1,14 @@
 'use strict'
 
 const Max7219 = require('..')
-const m = new Max7219({ device: '/dev/spidev0.0', controllerCount: 4, flip: 'vertical' })
+const m = new Max7219({ device: '/dev/spidev0.0', reverseDisplayOrdering:true, controllerCount: 4, flip: 'both' })
 
 async function init () {
-  await m.reset(0)
-  await m.reset(1)
-  await m.reset(2)
-  await m.reset(3)
+  //await m.reset(0)
+  //await m.reset(1)
+  //await m.reset(2)
+  //await m.reset(3)
+  await m.reset();
```

Comment:
no need to comment, this is git :) just delete the non-required lines! great!

Reply:
Examples updated.  Please review.